src/Interpreters/executeQuery.cpp

```diff
@@ -151,14 +152,22 @@ static void logQuery(const String & query, ContextPtr context, bool internal, Qu
         if (auto txn = context->getCurrentTransaction())
             transaction_info = fmt::format(" (TID: {}, TIDH: {})", txn->tid, txn->tid.getHash());
 
-        LOG_DEBUG(getLogger("executeQuery"), "(from {}{}{}){}{} {} (stage: {})",
-            client_info.current_address.toString(),
-            (current_user != "default" ? ", user: " + current_user : ""),
-            (!initial_query_id.empty() && current_query_id != initial_query_id ? ", initial_query_id: " + initial_query_id : std::string()),
-            transaction_info,
-            comment,
-            toOneLineQuery(query),
-            QueryProcessingStage::toString(stage));
+        std::istringstream iss(query);
+        std::string line;
+
+        for(UInt64 i = 0; std::getline(iss, line, '\n'); i++)
```

Comment:
I don't know if we will need this, can we just display line_number in one log message?

Reply:
Same for all below

Reply:
Yes, this is strange.

Reply:
updated